.npmrc

```diff
@@ -1,3 +1,3 @@
 strict-peer-dependencies=false
 enable-pre-post-scripts=true
-
+node-linker=hoisted
```

Comment:
This addresses the issue during build:
```
The inferred type of "X" cannot be named without a reference to "@polymorphic-factory/react"
```

Fix taken from https://github.com/microsoft/TypeScript/issues/42873#issuecomment-1481530011